src/main/kotlin/com/github/mdashl/rankedhelper/utility/Utility.kt

```diff
@@ -0,0 +1,17 @@
+package com.github.mdashl.rankedhelper.utility
+
+import com.github.mdashl.hypixel.elements.HypixelPlayer
+import com.github.mdashl.rankedhelper.playerSP
+import net.minecraft.util.ChatComponentText
+
+fun sendMessage(message: String): Unit = playerSP.addChatMessage(ChatComponentText(message))
```

Comment:
Rename playerSP to thePlayer and don't use = when method returns Unit.
```suggestion
fun sendMessage(message: String) {
    thePlayer.addChatMessage(ChatComponentText(message))
}
```